src/uct/ib/mlx5/dc/dc_mlx5.h

```diff
@@ -64,6 +64,7 @@ struct ibv_ravh {
 
 typedef struct uct_dc_mlx5_ep     uct_dc_mlx5_ep_t;
 typedef struct uct_dc_mlx5_iface  uct_dc_mlx5_iface_t;
+typedef uint16_t                  uct_dci_index_t;
```

Comment:
`_dc_mlx5_`

Reply:
I would prefer keeping it as short as possible, its being used in many places

Reply:
IMO would be too long and clumsy